### DISCUSSION TITLE
Can whisper output plaintext to a file without any formatting or timestamps?

### DISCUSSION THREAD

**Discussion by themani10** [openai/whisper#1754] [Q&A]
I used this code to try to get it to output .txt without timestamps but it still has the subtitling format:
```
model = whisper.load_model("tiny")
audio = whisper.load_audio(path)
result = model.transcribe(audio, bool=False)
writer = get_writer('txt', ".")
```
I want it to output plaintext without any formmatting. Any ideas?

**Comment by ryanheise** [comment]
Although you obtained a txt writer, you didn't use it. So any output you are seeing is not from the writer, and is probably just Whisper's on-screen display showing the transcription progress.

What you need to do is now invoke the writer function:

```python
writer(result, audio_path, {})
```

The output will be written to a file in the `"."` directory that you specified (as opposed to displaying on the screen).
